### PR TITLE
:shield:, pos_logout

### PR DESCRIPTION
**Pull request by fedoranvar**
Handled tour of module functionality;

### CODE REVIEW

File: pos_logout/README.rst [hunk 1]
```diff
@@ -10,6 +10,7 @@ Credits
 Contributors
 ------------
 * `Dinar Gabbasov <https://it-projects.info/team/GabbasovDinar>`__
+* `Kildebekov Anvar <https://it-projects.info/team/kildebekov>`__
```
Comment (KolushovAlexandr) [inline pos_logout/README.rst:14]: add me here too please

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]

**Comment by KolushovAlexandr** [comment]
Travis says there are wrong version numbers in readme
Add licences for test files

**Comment by fedoranvar** [comment]
> Travis says there are wrong version numbers in readme
> Add licences for test files

File: pos_logout/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,8 @@
+`1.0.1`
+-------
+
+- **Fix:** Handled tour: testing functional of pos_logout module by test user (created in python-part of test)
```
Comment (KolushovAlexandr) [inline pos_logout/doc/changelog.rst]: there is no need to change the version.
look through this 
https://gitlab.com/itpp/handbook/blob/master/documenting-updates.md
and this when and why we change the version 
https://odoo-development.readthedocs.io/en/latest/dev/docs/changelog.rst.html

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]

File: pos_logout/doc/changelog.rst (hunk 2 above)
Comment (KolushovAlexandr) [reply pos_logout/doc/changelog.rst]: the first link is about incorrect commit tag
it should be shield :shield:

**Review by KolushovAlexandr — APPROVED** [review]

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

File: pos_logout/tests/test_default.py [hunk 3]
```diff
@@ -18,6 +22,13 @@ def test_01_pos_is_loaded(self):
         # installed. In js web will only load qweb coming from modules
         # that are returned by the backend in module_boot. Without
         # this you end up with js, css but no qweb.
```
Comment (em230418) [inline pos_logout/tests/test_default.py:24]: Given link: https://github.com/odoo/odoo/blob/11.0/addons/point_of_sale/tests/test_frontend.py#L292-L297
is not correct, you need to use permalink with commit hash
https://github.com/odoo/odoo/blob/48dafd5b2011cee966920f664a904de2e2715ae8/addons/point_of_sale/tests/test_frontend.py#L306-L310

**Comment by em230418** [comment]
I approve to merge it now

**Pull request merged**